kytos/core/config.py

```diff
@@ -176,6 +176,10 @@ def _parse_options(self, argv):
             napps = options.napps_pre_installed
             options.napps_pre_installed = json.loads(napps)
 
+        # Gets vlan_pool after default is updated with values
+        # from config gile. then parses vlan_pool via json.loads
+        options.vlan_pool = json.loads(options.vlan_pool)
```

Comment:
Use the same check as in napps_pre_installed above.

```suggestion
        if isinstance(options.vlan_pool, str):
            options.vlan_pool = json.loads(options.vlan_pool)
```